internal/js/summary.js

```diff
@@ -1,413 +1,1392 @@
-var forEach = function (obj, callback) {
-  for (var key in obj) {
-    if (obj.hasOwnProperty(key)) {
-      if (callback(key, obj[key])) {
-        break
-      }
-    }
-  }
+/**
```

Comment:
for code reviews reason it would've been easier if the files weren't switched yet but done in a separate PR, as now I do not know if the other code was changed and how much and this diff here is pointless